src/commands/test.rs

```diff
@@ -345,6 +366,35 @@ pub async fn run_submission_tests(
                 "Score".if_supports_color(OwoStream::Stdout, |text| { text.bold() }),
                 score
             ));
+            let path = global.project.join("README.md");
```

Comment:
I think you'll have to locate the readme based on the pyproject. see https://docs.rs/pyproject-toml/latest/pyproject_toml/struct.Project.html#structfield.readme